### ISSUE TITLE
0.9.3 Influx cluster - Panic errors

### ISSUE THREAD

**Issue by anudharm** [influxdb/influxdb#4384]
Hi,

I have a 3 node influx cluster deployed with 0.9.3 version of influx. I intermittently keep getting 
```
{"log":"2015/10/09 15:15:56 http: panic serving 10.13.20.1:49799: runtime error: index out of range\n","stream":"stderr","time":"2015-10-09T15:15:56.244643762Z"}
{"log":"goroutine 36766 [running]:\n","stream":"stderr","time":"2015-10-09T15:15:56.244666946Z"}
{"log":"net/http.func·011()\n","stream":"stderr","time":"2015-10-09T15:15:56.244674698Z"}
{"log":"\u0009/root/.gvm/gos/go1.4.2/src/net/http/server.go:1130 +0xbb\n","stream":"stderr","time":"2015-10-09T15:15:56.244681356Z"}
{"log":"github.com/influxdb/influxdb/services/httpd.(*Handler).serveQuery(0xc2080102a0, 0x7f1ccb5d8910, 0xc20963bde0, 0xc20a3869c0, 0x0)\n","stream":"stderr","time":"2015-10-09T15:15:56.244688177Z"}
{"log":"\u0009/root/go/src/github.com/influxdb/influxdb/services/httpd/handler.go:289 +0x16c6\n","stream":"stderr","time":"2015-10-09T15:15:56.244694923Z"}
{"log":"github.com/influxdb/influxdb/services/httpd.*Handler.(github.com/influxdb/influxdb/services/httpd.serveQuery)·fm(0x7f1ccb5d8910, 0xc20963bde0, 0xc20a3869c0, 0x0)\n","stream":"stderr","time":"2015-10-09T15:15:56.244701355Z"}
{"log":"\u0009/root/go/src/github.com/influxdb/influxdb/services/httpd/handler.go:95 +0x4f\n","stream":"stderr","time":"2015-10-09T15:15:56.244708612Z"}
{"log":"github.com/influxdb/influxdb/services/httpd.func·001(0x7f1ccb5d8910, 0xc20963bde0, 0xc20a3869c0)\n","stream":"stderr","time":"2015-10-09T15:15:56.244715075Z"}
{"log":"\u0009/root/go/src/github.com/influxdb/influxdb/services/httpd/handler.go:621 +0x78\n","stream":"stderr","time":"2015-10-09T15:15:56.244721799Z"}
{"log":"net/http.HandlerFunc.ServeHTTP(0xc208159e60, 0x7f1ccb5d8910, 0xc20963bde0, 0xc20a3869c0)\n","stream":"stderr","time":"2015-10-09T15:15:56.244728485Z"}
{"log":"\u0009/root/.gvm/gos/go1.4.2/src/net/http/server.go:1265 +0x41\n","stream":"stderr","time":"2015-10-09T15:15:56.244734834Z"}
{"log":"github.com/influxdb/influxdb/services/httpd.func·002(0x7f1ccb5d8838, 0xc20963bd40, 0xc20a3869c0)\n","stream":"stderr","time":"2015-10-09T15:15:56.244741122Z"}
{"log":"\u0009/root/go/src/github.com/influxdb/influxdb/services/httpd/handler.go:679 +0x288\n","stream":"stderr","time":"2015-10-09T15:15:56.244747655Z"}
{"log":"net/http.HandlerFunc.ServeHTTP(0xc2080394b0, 0x7f1ccb5d8838, 0xc20963bd40, 0xc20a3869c0)\n","stream":"stderr","time":"2015-10-09T15:15:56.244754025Z"}
{"log":"\u0009/root/.gvm/gos/go1.4.2/src/net/http/server.go:1265 +0x41\n","stream":"stderr","time":"2015-10-09T15:15:56.244769309Z"}
{"log":"github.com/influxdb/influxdb/services/httpd.func·003(0x7f1ccb5d8838, 0xc20963bd40, 0xc20a3869c0)\n","stream":"stderr","time":"2015-10-09T15:15:56.244776095Z"}
{"log":"\u0009/root/go/src/github.com/influxdb/influxdb/services/httpd/handler.go:688 +0xba\n","stream":"stderr","time":"2015-10-09T15:15:56.244782604Z"}
{"log":"net/http.HandlerFunc.ServeHTTP(0xc208159e80, 0x7f1ccb5d8838, 0xc20963bd40, 0xc20a3869c0)\n","stream":"stderr","time":"2015-10-09T15:15:56.244789128Z"}
{"log":"\u0009/root/.gvm/gos/go1.4.2/src/net/http/server.go:1265 +0x41\n","stream":"stderr","time":"2015-10-09T15:15:56.244795336Z"}
{"log":"github.com/influxdb/influxdb/services/httpd.func·004(0x7f1ccb5d8838, 0xc20963bd40, 0xc20a3869c0)\n","stream":"stderr","time":"2015-10-09T15:15:56.244801537Z"}
{"log":"\u0009/root/go/src/github.com/influxdb/influxdb/services/httpd/handler.go:721 +0x31a\n","stream":"stderr","time":"2015-10-09T15:15:56.24480788Z"}
{"log":"net/http.HandlerFunc.ServeHTTP(0xc2080394e0, 0x7f1ccb5d8838, 0xc20963bd40, 0xc20a3869c0)\n","stream":"stderr","time":"2015-10-09T15:15:56.2448144Z"}
{"log":"\u0009/root/.gvm/gos/go1.4.2/src/net/http/server.go:1265 +0x41\n","stream":"stderr","time":"2015-10-09T15:15:56.244820914Z"}
{"log":"github.com/influxdb/influxdb/services/httpd.func·005(0x7f1ccb5d8838, 0xc20963bd40, 0xc20a3869c0)\n","stream":"stderr","time":"2015-10-09T15:15:56.244827114Z"}
{"log":"\u0009/root/go/src/github.com/influxdb/influxdb/services/httpd/handler.go:731 +0x179\n","stream":"stderr","time":"2015-10-09T15:15:56.244833449Z"}
{"log":"net/http.HandlerFunc.ServeHTTP(0xc208039500, 0x7f1ccb5d8838, 0xc20963bd40, 0xc20a3869c0)\n","stream":"stderr","time":"2015-10-09T15:15:56.244839882Z"}
{"log":"\u0009/root/.gvm/gos/go1.4.2/src/net/http/server.go:1265 +0x41\n","stream":"stderr","time":"2015-10-09T15:15:56.244846203Z"}
{"log":"github.com/influxdb/influxdb/services/httpd.func·006(0x7f1ccb5d8838, 0xc20963bd20, 0xc20a3869c0)\n","stream":"stderr","time":"2015-10-09T15:15:56.244852501Z"}
{"log":"\u0009/root/go/src/github.com/influxdb/influxdb/services/httpd/handler.go:739 +0x123\n","stream":"stderr","time":"2015-10-09T15:15:56.244859062Z"}
{"log":"net/http.HandlerFunc.ServeHTTP(0xc208159ea0, 0x7f1ccb5d8838, 0xc20963bd20, 0xc20a3869c0)\n","stream":"stderr","time":"2015-10-09T15:15:56.244865437Z"}
{"log":"\u0009/root/.gvm/gos/go1.4.2/src/net/http/server.go:1265 +0x41\n","stream":"stderr","time":"2015-10-09T15:15:56.244871947Z"}
{"log":"github.com/influxdb/influxdb/services/httpd.func·007(0x7f1ccb5d8800, 0xc209ed39a0, 0xc20a3869c0)\n","stream":"stderr","time":"2015-10-09T15:15:56.244879227Z"}
{"log":"\u0009/root/go/src/github.com/influxdb/influxdb/services/httpd/handler.go:749 +0x12c\n","stream":"stderr","time":"2015-10-09T15:15:56.244895698Z"}
{"log":"net/http.HandlerFunc.ServeHTTP(0xc208159ec0, 0x7f1ccb5d8800, 0xc209ed39a0, 0xc20a3869c0)\n","stream":"stderr","time":"2015-10-09T15:15:56.244902398Z"}
{"log":"\u0009/root/.gvm/gos/go1.4.2/src/net/http/server.go:1265 +0x41\n","stream":"stderr","time":"2015-10-09T15:15:56.244908676Z"}
{"log":"github.com/bmizerany/pat.(*PatternServeMux).ServeHTTP(0xc208066788, 0x7f1ccb5d8800, 0xc209ed39a0, 0xc20a3869c0)\n","stream":"stderr","time":"2015-10-09T15:15:56.244914824Z"}
{"log":"\u0009/root/go/src/github.com/bmizerany/pat/mux.go:109 +0x21c\n","stream":"stderr","time":"2015-10-09T15:15:56.244921446Z"}
{"log":"github.com/influxdb/influxdb/services/httpd.(*Handler).ServeHTTP(0xc2080102a0, 0x7f1ccb5d8800, 0xc209ed39a0, 0xc20a3869c0)\n","stream":"stderr","time":"2015-10-09T15:15:56.244927727Z"}
{"log":"\u0009/root/go/src/github.com/influxdb/influxdb/services/httpd/handler.go:167 +0x256\n","stream":"stderr","time":"2015-10-09T15:15:56.244934077Z"}
{"log":"net/http.serverHandler.ServeHTTP(0xc20b838a80, 0x7f1ccb5d8800, 0xc209ed39a0, 0xc20a3869c0)\n","stream":"stderr","time":"2015-10-09T15:15:56.244940196Z"}
{"log":"\u0009/root/.gvm/gos/go1.4.2/src/net/http/server.go:1703 +0x19a\n","stream":"stderr","time":"2015-10-09T15:15:56.244950992Z"}
{"log":"net/http.(*conn).serve(0xc209ed3900)\n","stream":"stderr","time":"2015-10-09T15:15:56.244957747Z"}
{"log":"\u0009/root/.gvm/gos/go1.4.2/src/net/http/server.go:1204 +0xb57\n","stream":"stderr","time":"2015-10-09T15:15:56.244964171Z"}
{"log":"created by net/http.(*Server).Serve\n","stream":"stderr","time":"2015-10-09T15:15:56.244981766Z"}
{"log":"\u0009/root/.gvm/gos/go1.4.2/src/net/http/server.go:1751 +0x35e\n","stream":"stderr","time":"2015-10-09T15:15:56.244989077Z"} errors.
```

**Comment by beckettsean** [comment]
Clustering in 0.9.3 was alpha. Please update to the latest 0.9.4.2 release.

**Comment by anudharm** [comment]
This seems to happen on a single host too.

**Comment by otoolep** [comment]
@anudharm -- this log format, while nice for machines, is difficult us to interpret. Can you please provide an unstructured log for us when reporting this issue?

**Comment by anudharm** [comment]
Here it is..

2015/10/16 22:11:58 http: panic serving 10.12.27.1:35240: runtime error: index out of range
goroutine 455 [running]:
net/http.func·011()
	/root/.gvm/gos/go1.4.2/src/net/http/server.go:1130 +0xbb
github.com/influxdb/influxdb/services/httpd.(*Handler).serveQuery(0xc2080103f0, 0x7f39087be4a8, 0xc208116220, 0xc2081384e0, 0x0)
	/root/go/src/github.com/influxdb/influxdb/services/httpd/handler.go:289 +0x16c6
github.com/influxdb/influxdb/services/httpd.*Handler.(github.com/influxdb/influxdb/services/httpd.serveQuery)·fm(0x7f39087be4a8, 0xc208116220, 0xc2081384e0, 0x0)
	/root/go/src/github.com/influxdb/influxdb/services/httpd/handler.go:95 +0x4f
github.com/influxdb/influxdb/services/httpd.func·001(0x7f39087be4a8, 0xc208116220, 0xc2081384e0)
	/root/go/src/github.com/influxdb/influxdb/services/httpd/handler.go:621 +0x78
net/http.HandlerFunc.ServeHTTP(0xc2080df620, 0x7f39087be4a8, 0xc208116220, 0xc2081384e0)
	/root/.gvm/gos/go1.4.2/src/net/http/server.go:1265 +0x41
github.com/influxdb/influxdb/services/httpd.func·002(0x7f39087be3d0, 0xc208116160, 0xc2081384e0)
	/root/go/src/github.com/influxdb/influxdb/services/httpd/handler.go:679 +0x288
net/http.HandlerFunc.ServeHTTP(0xc2080f15c0, 0x7f39087be3d0, 0xc208116160, 0xc2081384e0)
	/root/.gvm/gos/go1.4.2/src/net/http/server.go:1265 +0x41
github.com/influxdb/influxdb/services/httpd.func·003(0x7f39087be3d0, 0xc208116160, 0xc2081384e0)
	/root/go/src/github.com/influxdb/influxdb/services/httpd/handler.go:688 +0xba
net/http.HandlerFunc.ServeHTTP(0xc2080df640, 0x7f39087be3d0, 0xc208116160, 0xc2081384e0)
	/root/.gvm/gos/go1.4.2/src/net/http/server.go:1265 +0x41
github.com/influxdb/influxdb/services/httpd.func·004(0x7f39087be3d0, 0xc208116160, 0xc2081384e0)
	/root/go/src/github.com/influxdb/influxdb/services/httpd/handler.go:721 +0x31a
net/http.HandlerFunc.ServeHTTP(0xc2080f15f0, 0x7f39087be3d0, 0xc208116160, 0xc2081384e0)
	/root/.gvm/gos/go1.4.2/src/net/http/server.go:1265 +0x41
github.com/influxdb/influxdb/services/httpd.func·005(0x7f39087be3d0, 0xc208116160, 0xc2081384e0)
	/root/go/src/github.com/influxdb/influxdb/services/httpd/handler.go:731 +0x179
net/http.HandlerFunc.ServeHTTP(0xc2080f1610, 0x7f39087be3d0, 0xc208116160, 0xc2081384e0)
	/root/.gvm/gos/go1.4.2/src/net/http/server.go:1265 +0x41
github.com/influxdb/influxdb/services/httpd.func·006(0x7f39087be3d0, 0xc208116140, 0xc2081384e0)
	/root/go/src/github.com/influxdb/influxdb/services/httpd/handler.go:739 +0x123
net/http.HandlerFunc.ServeHTTP(0xc2080df660, 0x7f39087be3d0, 0xc208116140, 0xc2081384e0)
	/root/.gvm/gos/go1.4.2/src/net/http/server.go:1265 +0x41
github.com/influxdb/influxdb/services/httpd.func·007(0x7f39087be398, 0xc20813c140, 0xc2081384e0)
	/root/go/src/github.com/influxdb/influxdb/services/httpd/handler.go:749 +0x12c
net/http.HandlerFunc.ServeHTTP(0xc2080df680, 0x7f39087be398, 0xc20813c140, 0xc2081384e0)
	/root/.gvm/gos/go1.4.2/src/net/http/server.go:1265 +0x41
github.com/bmizerany/pat.(*PatternServeMux).ServeHTTP(0xc2080ceeb0, 0x7f39087be398, 0xc20813c140, 0xc2081384e0)
	/root/go/src/github.com/bmizerany/pat/mux.go:109 +0x21c
github.com/influxdb/influxdb/services/httpd.(*Handler).ServeHTTP(0xc2080103f0, 0x7f39087be398, 0xc20813c140, 0xc2081384e0)
	/root/go/src/github.com/influxdb/influxdb/services/httpd/handler.go:167 +0x256
net/http.serverHandler.ServeHTTP(0xc20813e360, 0x7f39087be398, 0xc20813c140, 0xc2081384e0)
	/root/.gvm/gos/go1.4.2/src/net/http/server.go:1703 +0x19a
net/http.(*conn).serve(0xc2080dcb40)
	/root/.gvm/gos/go1.4.2/src/net/http/server.go:1204 +0xb57
created by net/http.(*Server).Serve
	/root/.gvm/gos/go1.4.2/src/net/http/server.go:1751 +0x35e
2015/10/16 22:11:58 http: panic serving 10.12.27.1:35241: runtime error: index out of range
goroutine 484 [running]:
net/http.func·011()
	/root/.gvm/gos/go1.4.2/src/net/http/server.go:1130 +0xbb
github.com/influxdb/influxdb/services/httpd.(*Handler).serveQuery(0xc2080103f0, 0x7f39087be4a8, 0xc209d5a180, 0xc208176000, 0x0)
	/root/go/src/github.com/influxdb/influxdb/services/httpd/handler.go:289 +0x16c6
github.com/influxdb/influxdb/services/httpd.*Handler.(github.com/influxdb/influxdb/services/httpd.serveQuery)·fm(0x7f39087be4a8, 0xc209d5a180, 0xc208176000, 0x0)
	/root/go/src/github.com/influxdb/influxdb/services/httpd/handler.go:95 +0x4f
github.com/influxdb/influxdb/services/httpd.func·001(0x7f39087be4a8, 0xc209d5a180, 0xc208176000)
	/root/go/src/github.com/influxdb/influxdb/services/httpd/handler.go:621 +0x78
net/http.HandlerFunc.ServeHTTP(0xc2080df620, 0x7f39087be4a8, 0xc209d5a180, 0xc208176000)
	/root/.gvm/gos/go1.4.2/src/net/http/server.go:1265 +0x41
github.com/influxdb/influxdb/services/httpd.func·002(0x7f39087be3d0, 0xc209d5a0e0, 0xc208176000)
	/root/go/src/github.com/influxdb/influxdb/services/httpd/handler.go:679 +0x288
net/http.HandlerFunc.ServeHTTP(0xc2080f15c0, 0x7f39087be3d0, 0xc209d5a0e0, 0xc208176000)
	/root/.gvm/gos/go1.4.2/src/net/http/server.go:1265 +0x41
github.com/influxdb/influxdb/services/httpd.func·003(0x7f39087be3d0, 0xc209d5a0e0, 0xc208176000)
	/root/go/src/github.com/influxdb/influxdb/services/httpd/handler.go:688 +0xba
net/http.HandlerFunc.ServeHTTP(0xc2080df640, 0x7f39087be3d0, 0xc209d5a0e0, 0xc208176000)
	/root/.gvm/gos/go1.4.2/src/net/http/server.go:1265 +0x41
github.com/influxdb/influxdb/services/httpd.func·004(0x7f39087be3d0, 0xc209d5a0e0, 0xc208176000)
	/root/go/src/github.com/influxdb/influxdb/services/httpd/handler.go:721 +0x31a
net/http.HandlerFunc.ServeHTTP(0xc2080f15f0, 0x7f39087be3d0, 0xc209d5a0e0, 0xc208176000)
	/root/.gvm/gos/go1.4.2/src/net/http/server.go:1265 +0x41
github.com/influxdb/influxdb/services/httpd.func·005(0x7f39087be3d0, 0xc209d5a0e0, 0xc208176000)
	/root/go/src/github.com/influxdb/influxdb/services/httpd/handler.go:731 +0x179
net/http.HandlerFunc.ServeHTTP(0xc2080f1610, 0x7f39087be3d0, 0xc209d5a0e0, 0xc208176000)
	/root/.gvm/gos/go1.4.2/src/net/http/server.go:1265 +0x41
github.com/influxdb/influxdb/services/httpd.func·006(0x7f39087be3d0, 0xc209d5a0c0, 0xc208176000)
	/root/go/src/github.com/influxdb/influxdb/services/httpd/handler.go:739 +0x123
net/http.HandlerFunc.ServeHTTP(0xc2080df660, 0x7f39087be3d0, 0xc209d5a0c0, 0xc208176000)
	/root/.gvm/gos/go1.4.2/src/net/http/server.go:1265 +0x41
github.com/influxdb/influxdb/services/httpd.func·007(0x7f39087be398, 0xc2080800a0, 0xc208176000)
	/root/go/src/github.com/influxdb/influxdb/services/httpd/handler.go:749 +0x12c
net/http.HandlerFunc.ServeHTTP(0xc2080df680, 0x7f39087be398, 0xc2080800a0, 0xc208176000)
	/root/.gvm/gos/go1.4.2/src/net/http/server.go:1265 +0x41
github.com/bmizerany/pat.(*PatternServeMux).ServeHTTP(0xc2080ceeb0, 0x7f39087be398, 0xc2080800a0, 0xc208176000)
	/root/go/src/github.com/bmizerany/pat/mux.go:109 +0x21c
github.com/influxdb/influxdb/services/httpd.(*Handler).ServeHTTP(0xc2080103f0, 0x7f39087be398, 0xc2080800a0, 0xc208176000)
	/root/go/src/github.com/influxdb/influxdb/services/httpd/handler.go:167 +0x256
net/http.serverHandler.ServeHTTP(0xc20813e360, 0x7f39087be398, 0xc2080800a0, 0xc208176000)
	/root/.gvm/gos/go1.4.2/src/net/http/server.go:1703 +0x19a
net/http.(*conn).serve(0xc208080000)
	/root/.gvm/gos/go1.4.2/src/net/http/server.go:1204 +0xb57
created by net/http.(*Server).Serve
	/root/.gvm/gos/go1.4.2/src/net/http/server.go:1751 +0x35e
2015/10/16 22:11:58 http: panic serving 10.12.27.1:35242: runtime error: index out of range
goroutine 457 [running]:
net/http.func·011()
	/root/.gvm/gos/go1.4.2/src/net/http/server.go:1130 +0xbb
github.com/influxdb/influxdb/services/httpd.(*Handler).serveQuery(0xc2080103f0, 0x7f39087be4a8, 0xc208116220, 0xc20805a0d0, 0x0)
	/root/go/src/github.com/influxdb/influxdb/services/httpd/handler.go:289 +0x16c6
github.com/influxdb/influxdb/services/httpd.*Handler.(github.com/influxdb/influxdb/services/httpd.serveQuery)·fm(0x7f39087be4a8, 0xc208116220, 0xc20805a0d0, 0x0)
	/root/go/src/github.com/influxdb/influxdb/services/httpd/handler.go:95 +0x4f
github.com/influxdb/influxdb/services/httpd.func·001(0x7f39087be4a8, 0xc208116220, 0xc20805a0d0)
	/root/go/src/github.com/influxdb/influxdb/services/httpd/handler.go:621 +0x78
net/http.HandlerFunc.ServeHTTP(0xc2080df620, 0x7f39087be4a8, 0xc208116220, 0xc20805a0d0)
	/root/.gvm/gos/go1.4.2/src/net/http/server.go:1265 +0x41
github.com/influxdb/influxdb/services/httpd.func·002(0x7f39087be3d0, 0xc208116160, 0xc20805a0d0)
	/root/go/src/github.com/influxdb/influxdb/services/httpd/handler.go:679 +0x288
net/http.HandlerFunc.ServeHTTP(0xc2080f15c0, 0x7f39087be3d0, 0xc208116160, 0xc20805a0d0)
	/root/.gvm/gos/go1.4.2/src/net/http/server.go:1265 +0x41
github.com/influxdb/influxdb/services/httpd.func·003(0x7f39087be3d0, 0xc208116160, 0xc20805a0d0)
	/root/go/src/github.com/influxdb/influxdb/services/httpd/handler.go:688 +0xba
net/http.HandlerFunc.ServeHTTP(0xc2080df640, 0x7f39087be3d0, 0xc208116160, 0xc20805a0d0)
	/root/.gvm/gos/go1.4.2/src/net/http/server.go:1265 +0x41
github.com/influxdb/influxdb/services/httpd.func·004(0x7f39087be3d0, 0xc208116160, 0xc20805a0d0)
	/root/go/src/github.com/influxdb/influxdb/services/httpd/handler.go:721 +0x31a
net/http.HandlerFunc.ServeHTTP(0xc2080f15f0, 0x7f39087be3d0, 0xc208116160, 0xc20805a0d0)
	/root/.gvm/gos/go1.4.2/src/net/http/server.go:1265 +0x41
github.com/influxdb/influxdb/services/httpd.func·005(0x7f39087be3d0, 0xc208116160, 0xc20805a0d0)
	/root/go/src/github.com/influxdb/influxdb/services/httpd/handler.go:731 +0x179
net/http.HandlerFunc.ServeHTTP(0xc2080f1610, 0x7f39087be3d0, 0xc208116160, 0xc20805a0d0)
	/root/.gvm/gos/go1.4.2/src/net/http/server.go:1265 +0x41
github.com/influxdb/influxdb/services/httpd.func·006(0x7f39087be3d0, 0xc208116140, 0xc20805a0d0)
	/root/go/src/github.com/influxdb/influxdb/services/httpd/handler.go:739 +0x123
net/http.HandlerFunc.ServeHTTP(0xc2080df660, 0x7f39087be3d0, 0xc208116140, 0xc20805a0d0)
	/root/.gvm/gos/go1.4.2/src/net/http/server.go:1265 +0x41
github.com/influxdb/influxdb/services/httpd.func·007(0x7f39087be398, 0xc208080460, 0xc20805a0d0)
	/root/go/src/github.com/influxdb/influxdb/services/httpd/handler.go:749 +0x12c
net/http.HandlerFunc.ServeHTTP(0xc2080df680, 0x7f39087be398, 0xc208080460, 0xc20805a0d0)
	/root/.gvm/gos/go1.4.2/src/net/http/server.go:1265 +0x41
github.com/bmizerany/pat.(*PatternServeMux).ServeHTTP(0xc2080ceeb0, 0x7f39087be398, 0xc208080460, 0xc20805a0d0)
	/root/go/src/github.com/bmizerany/pat/mux.go:109 +0x21c
github.com/influxdb/influxdb/services/httpd.(*Handler).ServeHTTP(0xc2080103f0, 0x7f39087be398, 0xc208080460, 0xc20805a0d0)
	/root/go/src/github.com/influxdb/influxdb/services/httpd/handler.go:167 +0x256
net/http.serverHandler.ServeHTTP(0xc20813e360, 0x7f39087be398, 0xc208080460, 0xc20805a0d0)
	/root/.gvm/gos/go1.4.2/src/net/http/server.go:1703 +0x19a
net/http.(*conn).serve(0xc20813c140)
	/root/.gvm/gos/go1.4.2/src/net/http/server.go:1204 +0xb57
created by net/http.(*Server).Serve
	/root/.gvm/gos/go1.4.2/src/net/http/server.go:1751 +0x35e
2015/10/16 22:11:58 http: panic serving 10.12.27.1:35243: runtime error: index out of range

**Comment by otoolep** [comment]
Can you please upgrade to 0.9.4.2? There have been lots of fixes since 0.9.3. If you still hit the issue with the new version, we can re-open.

https://influxdb.com/download/index.html
